style.css

```diff
@@ -198,6 +202,9 @@ ol, ul, li, p {
 li, p {
   margin-left: 0;
 }
+address {
+  font-style: normal;
+}
 
 .github-fork-ribbon-wrapper {
   display: none;
```

Comment:
Yeah, xml2rfc still produces braindead HTML for the toc.  And you use `<address>`, which causes address to be italicized in many browsers, against all good taste.